### DISCUSSION TITLE
Revamping our repo: Closing over 75% of our issues with these changes

### DISCUSSION THREAD

**Discussion by JesusPaz** [nodesource/distributions#1625] [General]
Hello everyone,

We're well aware that our repository support hasn't been as active as we would have liked, but we come bearing news. Starting next week, we'll be rolling out a more modern version of our repository. We've taken a close look at the most common issues and worked diligently to address them by updating all of our internal processes to ensure a better result for all of you.

In the coming week, we will be releasing an official post outlining the entire migration process and providing instructions on how to configure this new version of the repository. Stay tuned for more exciting news!

Best regards,
NodeSource Team

**Comment by JohnXLivingston** [comment]
Good news if that closes 75% of your issues!

For the record, the `/etc/apt/keyrings` folder does not exist on some distro (for example Debian Bullseye, was only added in Bookwork). So the procedure will fail, adding a `mkdir -p /etc/apt/keyrings` should do the trick.

**Comment by blueray453** [comment]
Is there any way to get a notification with the link to the new repository, when you will be rolling out a more modern version of the repository.
